pymc3/step_methods/hmc/nuts.py

```diff
@@ -90,7 +91,8 @@ class NUTS(BaseHMC):
     def __init__(self, vars=None, Emax=1000, target_accept=0.8,
                  gamma=0.05, k=0.75, t0=10, adapt_step_size=True,
                  max_treedepth=10, on_error='summary', **kwargs):
-        R"""
+        R"""Set up the No-U-Turn sampler.
```

Comment:
The standard says that we shouldn't document the arguments in `__init__` and in the class doc string.
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#class-docstring